index.html

```diff
@@ -0,0 +1,293 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>
+      Digital Credentials API
```

Comment:
I personally think that the term Digital Credentials is the best that we could use, but I hear that this isn't something particularly shared by everyone. Maybe we could put up for discussion in a issue and rename it as a separate step?

Reply:
Was there a reason to not use the "Identity Credential" terminology here?

Reply:
Apologies I didn't see #55 had been raised.